bin/import-test-content.php

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/php
+<?php
+
+namespace WPOrg_Learn\Bin\ImportTestContent;
+
+/**
+ * CLI script for generating local test content, fetched from the live learn.wordpress.org site.
+ *
+ * This needs to be run in a wp-env, for example:
+ *
+ * yarn run wp-env run cli "php bin/import-test-content.php"
+ */
+
+// This script should only be called in a CLI environment.
+if ( 'cli' != php_sapi_name() ) {
+    die();
+}
+
+
+$opts = getopt( '', array( 'post:', 'url:', 'abspath:', 'age:' ) );
+
+require dirname( dirname( __FILE__ ) ) . '/wp-load.php';
+
+// Refuse to run on a production site.
+$run_on_hosts = [
+	'localhost',
+	'127.0.0.1',
+];
+if ( !in_array( parse_url( get_site_url(), PHP_URL_HOST ), $run_on_hosts ) ) {
```

Comment:
```suggestion
if ( 'local' !== wp_get_environment_type() ) {
```

This might be more consistent with how our other codebases do similar checks.